lib/tests/CMakeLists.txt

```diff
@@ -106,7 +106,7 @@ include_directories(
 )
 
 add_executable(libfacter_test $<TARGET_OBJECTS:libfactersrc> ${LIBFACTER_TESTS_COMMON_SOURCES} ${LIBFACTER_TESTS_PLATFORM_SOURCES} ${LIBFACTER_TESTS_CATEGORY_SOURCES})
-set_target_properties(libfacter_test PROPERTIES COTIRE_UNITY_LINK_LIBRARIES_INIT "COPY_UNITY" COTIRE_ENABLE_PRECOMPILED_HEADER ${PRECOMPILED_HEADERS} COTIRE_ADD_UNITY_BUILD FALSE)
+set_target_properties(libfacter_test PROPERTIES COTIRE_UNITY_LINK_LIBRARIES_INIT "COPY_UNITY" COTIRE_ENABLE_PRECOMPILED_HEADER ${PRECOMPILED_HEADERS} COTIRE_ADD_UNITY_BUILD FALSE COTIRE_PREFIX_HEADER_IGNORE_PATH "${LEATHERMAN_CATCH_INCLUDE}")
```

Comment:
Now that catch is no longer part of our source tree, we need to explicitly exclude it from PCH (since we `#include` it with different preprocessor definitions set)